internal/subscriber/subscriberimpl_test.go

```diff
@@ -250,7 +253,7 @@ func getMockSubscriber(ctx context.Context, subImpl *BasicImplementation) *Subsc
 		closeChan:           make(chan struct{}),
 		modAckChan:          make(chan *ModAckMessage, 10),
 		deadlineTicker:      time.NewTicker(1 * time.Second),
-		healthMonitorTicker: time.NewTicker(1 * time.Minute),
+		healthMonitorTicker: time.NewTicker(1 * time.Second),
```

Comment:
why have we changed this..?

Reply:
This will increase coverage and test healthMonitoring is working fine or not

Reply:
Increasing test coverage is not about reaching a particular no. The more coverage we have, less are the chances of committing buggy code.
Changing this health check ticker monitor has no impact on the latter, but will only increase some numbers on paper.

Reply:
yes exactly

Reply:
and also if there is some error in monitoring code this fail the test case

Reply:
I don't see any downside here

Reply:
@sumitkrzp Please try to figure out the reason why ticker interval change inc the code coverage in the first case, and then we will see, if it really make sense to do the change.

Reply:
@sakshi-goyal-razorpay 
This will take care of bellow lines : 
`case <-s.healthMonitorTicker.C: 
lag := s.subscriberImpl.GetConsumerLag()
			for topic, offset := range lag {
				tp := strings.Split(topic, "=")
				if len(tp) == 1 {
					logger.Ctx(ctx).Errorw("subscriber: failed to parse topic and partition for consumer lag", "topic", topic)
					continue
				}
				subscriberPartitionConsumerLag.WithLabelValues(env, tp[0], s.subscription.Name, tp[1]).Set(float64(offset))
			}`